terraform/modules/hub/hub_frontend.tf

```diff
@@ -36,6 +36,11 @@ resource "aws_security_group_rule" "frontend_task_egress_to_internet_over_https"
 
 locals {
   location_blocks = <<-LOCATIONS
+  # Reject DEBUG verb in all cases.
+  if ($request_method = DEBUG ) {
```

Comment:
Yes, the single `=` for comparison [is correct](http://nginx.org/en/docs/http/ngx_http_rewrite_module.html#if).

Reply:
There's an extra whitespace after `DEBUG` though.

Reply:
```suggestion
  if ($request_method = DEBUG) {
```